Tests/TestCommon.m

```diff
@@ -91,8 +91,6 @@ - (instancetype)init
 - (void)dealloc
 {
 #ifdef DELETE_STORE_AFTER_EACH_TEST_METHOD
```

Comment:
DELETE_STORE_AFTER_EACH_TEST_METHOD should now be defined by default or removed. @ericwa, what's your take on this?